simtools/util/names.py

```diff
@@ -343,7 +344,7 @@ def convertTelescopeNameToYaml(name):
     'Trigger': ['trigger']
 }
 
-allArrayNames = {
+allLayoutArrayNames = {
```

Comment:
Need to discuss if we want to have all these hardwired lists.

Reply:
Probably not. This should go into a global config file. Keeping these names here was a temporary solution that is very easy to fix.